### PR TITLE
World replies back!

### PR DESCRIPTION
**Pull request by alyfarahat**
Just for fun!

### CODE REVIEW

File: hello.py [hunk 1]
```diff
@@ -1,3 +1,8 @@
 # hello world, again
 
+def worldRepliesBack():
+	print 'Hello, boy!'
```
Comment (cswiercz) [inline hello.py]: This should read `Hello, right back!`, instead.

**Comment by cswiercz** [comment]
@alyfarahat Mind if, during Tuesday's lecture, I use your contribution to demonstrate how to merge someone else's code?

**Comment by alyfarahat** [comment]
@cswiercz, sure! Please feel free to use it...

**Comment by cswiercz** [comment]
Excellent. I'll leave it un-merged for now.

**Pull request merged**